modules/logging-es-deploy-cli.adoc

```diff
@@ -0,0 +1,319 @@
+----
+$ oc apply -f <filename>.yaml
+----
+
+. Create a `ClusterLogging` object as a YAML file:
++
+--
+.Example `ClusterLogging` object
+[source,yaml]
+----
+apiVersion: logging.openshift.io/v1
+kind: ClusterLogging
+metadata:
+  name: instance # <1>
+  namespace: openshift-logging
+spec:
+  managementState: Managed # <2>
+  logStore:
+    type: elasticsearch # <3>
+    retentionPolicy: # <4>
+      application:
+        maxAge: 1d
+      infra:
+        maxAge: 7d
+      audit:
+        maxAge: 7d
+    elasticsearch:
+      nodeCount: 3 # <5>
+      storage:
+        storageClassName: <storage_class_name> # <6>
+        size: 200G
+      resources: # <7>
+          limits:
+            memory: 16Gi
+          requests:
+            memory: 16Gi
+      proxy: # <8>
+        resources:
+          limits:
+            memory: 256Mi
+          requests:
+            memory: 256Mi
+      redundancyPolicy: SingleRedundancy
+  visualization:
+    type: kibana # <9>
+    kibana:
+      replicas: 1
+  collection:
+    type: fluentd # <10>
+    fluentd: {}
+----
+<1> The name must be `instance`.
+<2> The OpenShift Logging management state. In some cases, if you change the OpenShift Logging defaults, you must set this to `Unmanaged`.
+However, an unmanaged deployment does not receive updates until OpenShift Logging is placed back into a managed state.
+<3> Settings for configuring Elasticsearch. Using the CR, you can configure shard replication policy and persistent storage.
+<4> Specify the length of time that Elasticsearch should retain each log source. Enter an integer and a time designation: weeks(w), hours(h/H), minutes(m) and seconds(s). For example, `7d` for seven days. Logs older than the `maxAge` are deleted. You must specify a retention policy for each log source or the Elasticsearch indices will not be created for that source.
+<5> Specify the number of Elasticsearch nodes.
+<6> Enter the name of an existing storage class for Elasticsearch storage. For best performance, specify a storage class that allocates block storage. If you do not specify a storage class, OpenShift Logging uses ephemeral storage.
+<7> Specify the CPU and memory requests for Elasticsearch as needed. If you leave these values blank, the OpenShift Elasticsearch Operator sets default values that should be sufficient for most deployments. The default values are `16Gi` for the memory request and `1` for the CPU request.
+<8> Specify the CPU and memory requests for the Elasticsearch proxy as needed. If you leave these values blank, the OpenShift Elasticsearch Operator sets default values that should be sufficient for most deployments. The default values are `256Mi` for the memory request and `100m` for the CPU request.
+<9> Settings for configuring Kibana. Using the CR, you can scale Kibana for redundancy and configure the CPU and memory for your Kibana nodes.
+<10> Settings for configuring Fluentd. Using the CR, you can configure Fluentd CPU and memory limits.
```

Comment:
If it's changed the collector "type: vector", then, it shouldn't be needed this reference to Fluentd also